src/main/java/bio/terra/common/kubernetes/KubePodListener.java

```diff
@@ -177,14 +210,10 @@ Exception getException() {
     return podMap;
   }
 
+  // get the number of running pods from the podMap
   int getActivePodCount() {
-    int count = 0;
-    for (PodState podState : podMap.values()) {
-      if (podState == PodState.RUNNING) {
-        count++;
-      }
-    }
+    final long count = podMap.values().stream().filter(PodState.RUNNING::equals).count();
     logger.info("KubePodListener ActivePodCount: {} of {} pods active", count, podMap.size());
-    return count;
+    return (int) count;
```

Comment:
Maybe use Math.toIntExact instead of the cast.

Reply:
oh nice